src/test/kotlin/solve/parsers/CSVParserTestsUtils.kt

```diff
@@ -0,0 +1,10 @@
+package solve.parsers.lines
+
+import java.io.File
+
+internal fun createCSVFileWithData(tempFolder: File, csvStringData: String): File {
```

Comment:
Move test utils into the test package

Reply:
> Move test utils into the test package

You mean `src/test`? I think now it is cleaner